### ISSUE TITLE
How do we enter "admin" mode for a competition

### ISSUE THREAD

**Issue by olyerickson** [codalab/codalab-competitions#3117]
We're trying to access "Admin" mode for a selected competition. 

I'm logged-in and I'm an admin, but the UI really only looks the same as a regular logged-in user. 

(I remember asking Adrian about this years ago but don't remember the answer. Also, I could find it in the README)

**Comment by olyerickson** [comment]
The above recommends going to the `codalab-competitions` folder on the machine running the instance, but we have no idea where that is...

**Comment by olyerickson** [comment]
Okay, we found... `/home/codalab/codalab-competitions` on the host machine...

It's still unclear to us what "Delete all contents of database" means...

**Comment by Didayolo** [comment]
Hello John,

_I've replied to your email. I paste the answer here in case it can be useful._

### How to restart a competition?

I guess the easiest way is simply to **re-upload the competition from scratch**. This way you have a new competition with a clear leaderboard and the possibility for new participants to upload new competitions.
You may need to change the starting / ending dates using the editor.


### What does "delete all contents of database" mean?

Deleting all content from the database is a platform admin procedure (not a competition organizer one). What it does is erases the content from the platform database, which contains **all user accounts, information about competitions, etc.**
Basically if you do that, you don't restart one competition, instead you "restart" (or re-instate) the whole platform. **So be careful with that!**

The `codalab-competitions` folder is the root of the project on the web server serving your CodaLab platform.

I hope this helps.
